addons/mail/static/src/models/thread/thread.js

```diff
@@ -636,6 +636,7 @@ registerModel({
                 attachments: attachmentsData,
                 followers: followersData,
                 suggestedRecipients: suggestedRecipientsData,
+                hasAccess: hasAccess,
```

Comment:
don't need to repeat the key when it has the same value twice, (same remark below)